### PR TITLE
Quantiles: Clarify to use type 7

### LINKED ISSUE #296: Which quantile type?
Related: while looking up example definitions and descriptions I stumbled on this:
https://blogs.sas.com/content/iml/2017/05/24/definitions-sample-quantiles.html referencing https://www.amherst.edu/media/view/129116/original/Sample+Quantiles.pdf

> there are nine definitions of sample quantiles that commonly appear in statistical software packages. 

If we want consistency across backends, we probably have to pick which flavor of  "sample quantiles"  we want.

at the moment I  have no idea which flavor that's currently used in VITO/EODC backends for example

_Originally posted by @soxofaan in https://github.com/Open-EO/openeo-processes/issues/294#issuecomment-952671519_

### PR DESCRIPTION
**Pull request by m-mohr**
Fixes #296 

From what I read in #296, it seems only VITO would need to switch from "Legacy" to R_7 in their library and Sinergise would need to switch from "higher" to "linear". All other implementations seem to implement type 7 already.

I've not chosen to go for a list yet and only clarify the type for now. If there's demand for more types, we can add that later.

### CODE REVIEW

File: meta/implementation.md [hunk 1]
```diff
@@ -141,3 +141,17 @@ To make `date_shift` easier to implement, we have found some libraries that foll
...
+## Quantile algorithms
+
+The `quantiles` could implement a number of different algorithms, literature usually distinguishes [9 types](https://en.wikipedia.org/wiki/Quantile#Estimating_quantiles_from_a_sample).
+Right now it's not possible to choose from them, but it may be added in the future.
+To improve interoperability openEO processes v1.2 added details about the algorithm that should be implemented.
+A survey has shown that most libraries implement type 7 and as such this was chosen to be the default.
+
+We have found some libraries that can be used for an implementation:
+- Java: [Apache Commons Math Percentile](http://commons.apache.org/proper/commons-math/javadocs/api-3.6/org/apache/commons/math3/stat/descriptive/rank/Percentile.html), choose the [estimation type `R_7`](http://commons.apache.org/proper/commons-math/javadocs/api-3.6/org/apache/commons/math3/stat/descriptive/rank/Percentile.EstimationType.html#R_7)
+- JavaScript: [d3](https://github.com/d3/d3-array/blob/v2.8.0/README.md#quantile), implements type 7 only.
```
Comment (dthiex) [inline meta/implementation.md:154]: Change wording for better understanding --> has only type 7 implemented

**Review by dthiex — CHANGES_REQUESTED** [review]
Besides the small comment changes seem clear and understandable.

File: meta/implementation.md (hunk 1 above)
Comment (m-mohr) [reply meta/implementation.md]: Thanks, done.

**Review by dthiex — APPROVED** [review]

File: meta/implementation.md [hunk 2]
```diff
@@ -141,3 +141,17 @@ To make `date_shift` easier to implement, we have found some libraries that foll
 - JavaScript: [Moment.js](https://momentjs.com/)
 - Python: [dateutil](https://dateutil.readthedocs.io/en/stable/index.html)
 - R: [lubridate](https://lubridate.tidyverse.org/) ([Cheatsheet](https://rawgit.com/rstudio/cheatsheets/master/lubridate.pdf))
+
+## Quantile algorithms
+
+The `quantiles` could implement a number of different algorithms, literature usually distinguishes [9 types](https://en.wikipedia.org/wiki/Quantile#Estimating_quantiles_from_a_sample).
+Right now it's not possible to choose from them, but it may be added in the future.
+To improve interoperability openEO processes v1.2 added details about the algorithm that should be implemented.
```
Comment (m-mohr) [inline meta/implementation.md:149]:
```suggestion
To improve interoperability openEO processes, version 1.2.0 added details about the algorithm that must be implemented.
```

File: quantiles.json [hunk 3]
```diff
@@ -173,6 +173,12 @@
             "rel": "about",
             "href": "https://en.wikipedia.org/wiki/Quantile",
             "title": "Quantiles explained by Wikipedia"
+        },
+        {
+            "rel": "about",
+            "href": "https://www.amherst.edu/media/view/129116/original/Sample+Quantiles.pdf",
+            "type": "application/pdf",
+            "title": "Hyndman and Fan (1996): 'Sample Quantiles in Statistical Packages'"
```
Comment (m-mohr) [inline quantiles.json:181]:
```suggestion
            "title": "Hyndman and Fan (1996): Sample Quantiles in Statistical Packages"
```

**Comment by m-mohr** [comment]
Last chance to review, I plan to merge this on Wednesday!

**Comment by clausmichele** [comment]
In quantiles.json: at the end of description there is S, did you mean JS?

**Comment by m-mohr** [comment]
@clausmichele No, it refers to the programming language S :)

**Comment by clausmichele** [comment]
> 

Thanks, I've learned something new today :)

**Review by clausmichele — APPROVED** [review]

**Pull request merged**